src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml

```diff
@@ -475,7 +475,37 @@
                     </div>
                 </li>
             }
-        </ul>             
+        </ul>
+        <p class="h6">
+            <a data-bs-toggle="collapse" href="#apiRevisionOptionsCollapse" aria-expanded="true" aria-controls="approvalCollapse">API Revision Options&nbsp;&nbsp;<i class="fa-solid fa-ellipsis"></i></a>

```

Comment:
Can you change aria-controls value to `apiRevisionOptionsCollapse`. aria-controld value should be the same as the href without the '#'

Could you also update these lines
https://github.com/Azure/azure-sdk-tools/blob/115b3f0f30b4a96d15567ab03bee0b4f7354957a/src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml#L334

https://github.com/Azure/azure-sdk-tools/blob/115b3f0f30b4a96d15567ab03bee0b4f7354957a/src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml#L369

Reply:
Made 3 changes